packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -77,9 +82,21 @@ const shouldShowCounter = computed(() => props.withCounter && props.maxlength);
     <label>
       <div
         v-if="!props.hideLabel"
-        :class="['onyx-input__label', 'onyx-text--small', requiredMarkerClass]"
+        class="onyx-input__label onyx-text--small"
+        :class="[!props.required ? requiredMarkerClass : undefined]"
       >
-        <div class="onyx-truncation-ellipsis">{{ props.label }}</div>
+        <div class="onyx-input__header">
+          <div class="onyx-truncation-ellipsis">
+            {{ props.label }}
+          </div>
+          <div :class="[props.required ? requiredMarkerClass : undefined]"></div>
+          <OnyxTooltip v-if="props.labelTooltip" open="hover" :text="props.labelTooltip">
+            <button :aria-label="t('infoTooltip')" class="onyx-input__tooltip-trigger">
+              <OnyxIcon :icon="circleInformation" color="neutral" size="12px" />
+            </button>
+          </OnyxTooltip>
+          <div class="onyx-input__optional">{{ t("optional") }}</div>
```

Comment:
```suggestion
        <div class="onyx-input__header">
          <span class="onyx-truncation-ellipsis">
            {{ props.label }}
          </span>
          <span :class="[props.required ? requiredMarkerClass : undefined]"></span>
          <OnyxTooltip v-if="props.labelTooltip" open="hover" :text="props.labelTooltip">
            <button :aria-label="t('infoTooltip')" class="onyx-input__tooltip-trigger">
              <OnyxIcon :icon="circleInformation" color="neutral" size="12px" />
            </button>
          </OnyxTooltip>
          <span class="onyx-input__optional">{{ t("optional") }}</span>
```

Maybe we can use `<span>` instead so we don't have divs everywhere :D